src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -180,55 +176,42 @@
                        view-finder (assoc layout :height (:width layout))]
                    (reset! qr-view-finder view-finder)))}])
 
-(defn- border
-  [border1 border2 corner]
-  [rn/view {:style (style/border border1 border2 corner)}])
+(defn- white-border
+  [corner]
+  (let [border-styles (style/white-border corner)]
+    [rn/view
+     [rn/view {:style (border-styles :border)}]
+     [rn/view {:style (border-styles :tip-1)}]
+     [rn/view {:style (border-styles :tip-2)}]]))
 
-(defn- border-tip
-  [{:keys [top bottom left right]}]
-  [rn/view
-   {:style (style/border-tip top bottom right left)}])
+(defn- white-square
+  [layout-size]
+  [rn/view {:style (style/qr-view-finder-container layout-size)}
+   [rn/view {:style style/view-finder-border-container}
+    [white-border :top-left]
+    [white-border :top-right]]
+   [rn/view {:style style/view-finder-border-container}
+    [white-border :bottom-left]
+    [white-border :bottom-right]]])
 
 (defn- viewfinder
   [qr-view-finder]
-  (let [size (+ (:width qr-view-finder) 2)]
-    [:<>
-     [rn/view {:style (style/viewfinder-container qr-view-finder)}
-      [rn/view
-       {:style (style/qr-view-finder-container size)}
-       [rn/view
-        {:style style/view-finder-border-container}
-        [rn/view
-         [border :border-top-width :border-left-width :border-top-left-radius]
-         [border-tip {:right -1 :top 0}]
-         [border-tip {:left 0 :bottom -1}]]
-        [rn/view
-         [border :border-top-width :border-right-width :border-top-right-radius]
-         [border-tip {:right 0 :bottom -1}]
-         [border-tip {:left -1 :top 0}]]]
-       [rn/view {:flex-direction :row :justify-content :space-between}
-        [rn/view
-         [border :border-bottom-width :border-left-width :border-bottom-left-radius]
-         [border-tip {:right -1 :bottom 0}]
-         [border-tip {:left 0 :top -1}]]
-        [rn/view
-         [border :border-bottom-width :border-right-width :border-bottom-right-radius]
-         [border-tip {:right 0 :top -1}]
-         [border-tip {:left -1 :bottom 0}]]]]
-      [quo/text
-       {:size   :paragraph-2
-        :weight :regular
-        :style  style/viewfinder-text}
-       (i18n/label :t/ensure-qr-code-is-in-focus-to-scan)]]]))
+  (let [layout-size (+ (:width qr-view-finder) 2)]
```

Comment:
I'd destructure width on line 198

Reply:
I think it's not worth it in this case.
Just a key extraction, maybe if there are more keys, it'd be better to destructure. I think both are fine